README.md

```diff
@@ -104,23 +96,24 @@ The following limitations will be addressed in future releases (see [Roadmap](#r
 - [x] Dockerfile
 - [x] github actions
 - [x] metrics
-- [ ] support multiple matchers
+- [x] support multiple matchers
 - [ ] support empty matchers
 - [ ] support store-API for better performance
 - [ ] return error message instead of only logging it (??)
 - [ ] remove space after comma in metrics (causing no issues)
 
 ## Metrics
 
-| Metric                                              | Notes
-|-----------------------------------------------------|------------------------------------
-| thanosfederateproxy_scrape_duration_seconds_count   | Total number of scrape requests with response code
-| thanosfederateproxy_scrape_duration_seconds_sum     | Duration of scrape requests with response code
-| thanosfederateproxy_scrape_duration_seconds_bucket  | Count of scrape requests per bucket (for calculating percentile)
+| Metric                                             | Notes                                                            |
+| -------------------------------------------------- | ---------------------------------------------------------------- |
```

Comment:
These were automatically formatted by a markdown linter.

Reply:
I can apply these changes but if you're planning on having a linter in the CI I think it's better to keep it as it is as these were made by a linter...

Reply:
This is the format I used to write tables, and it was because of my IDE recommendation, but as I checked it out, the format you used is correct. thanks for your comments, I'll resolve this thread